pint/models/parameter.py

```diff
@@ -134,6 +138,17 @@ def value(self,val):
                     raise ValueError("The ._num_value has to be a pure number or None. "\
                                      "Please check your .get_num_value method. ")
 
+    def prior_probability(self,value=None):
+        """Return the prior probability, evaluated at the current value of
+        the parameter, or at a proposed value.
+        
+        Probabilities are evaluated using the num_value attribute
+        """
+        if value is None:
+            return self.prior.prior_probability(self.num_value)
+        else:
+            return self.prior.prior_probability(value)
+            
```

Comment:
A slightly different (and perhaps a bit more pythonic) way to do something like this would be:

``
val = self.num_value if value is None else value
return self.prior.prior_probability(val)
``
With a shorter method name, it could easily be a one-liner:
``
return self.pp(self.num_value) if value is None else self.pp(value)
``
Not a big deal, though...